_episodes/05-memory.md

```diff
@@ -243,7 +243,7 @@ Cognitive load theory's proponents claim that eliminating extraneous cognitive l
 way to do this is by using *guided practice*: we set up a structure in which learners can
 apply their skills with direction and get feedback on their progress.
 
-> ## Why We Don't Use Minimal Guidance
+> ## Why We Do not Use Minimal Guidance
```

Comment:
should "not" be capitalized?

Reply:
I was curious so I did a quick search... it seems like there isn't one decision on this.  If we want to be consistent somehow, we could pick and put it in the main style guide for the Carpentries on which flavour of title case we prefer.  My vote is that "do" and "not" are in the same case. 

https://www.thoughtco.com/which-words-in-a-title-should-be-capitalized-1691026

(PS. i'm very proud of myself for not using the US spelling of flavour, even though my browser is yelling at me for spelling it "incorrectly")

Reply:
Agree that do + not are either both upper or lower case.